### PR TITLE
Add basic components package readme

### PR DESCRIPTION
**Pull request by SiAdcock**
## What is the purpose of this change?

The components package readme needs some rudimentary introduction to the package.

## What does this change?

-   Adds a stripped-down readme

## Is this it?

It's likely that this will tie in to the work on Storybook documentation reorganisation. We could either leave this as a starter for 10, or implement this README as part of the wider docs work.

### CODE REVIEW

File: packages/@guardian/source-react-components/README.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 [![npm](https://img.shields.io/npm/v/@guardian/source-react-components)](https://www.npmjs.com/package/@guardian/source-react-components)
 
-⚠️🚧 _UNDER CONSTRUCTION You probably want one of the [`@guardian/src-*`](https://www.npmjs.com/search?q=%40guardian%2Fsrc-) packages for the moment._ 🚧⚠️
+Components are individually installable as JavaScript packages via NPM or Yarn. Each component package exposes its React components, themes and TypeScript types.
```
Comment (sndrs) [inline packages/@guardian/source-react-components/README.md:5]:
```suggestion
A set of robust, accessible React components built using `@guardian/source-foundations`.

See [the `@guardian/source-react-components` storybook](https://guardian.github.io/source/?path=/docs/Packages/source-react-components) for full info.
```

File: packages/@guardian/source-react-components/README.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+$ yarn add @guardian/source-react-components
+```
+
+or
+
+```sh
+$ npm install @guardian/source-react-components
+```
+
+## Use
```
Comment (sndrs) [inline packages/@guardian/source-react-components/README.md:19]:
```suggestion
## Example
```

File: packages/@guardian/source-react-components/README.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+
+```tsx
+import { TextInput, Button } from '@guardian/source-react-components';
+
+const Form = () => (
+    <form>
+        <TextInput label="First name" />
+        <Button>Submit</Button>
+    </>
```
Comment (sndrs) [inline packages/@guardian/source-react-components/README.md:28]:
```suggestion
    </form >
```

File: packages/@guardian/source-react-components/README.md (hunk 1 above)
Comment (SiAdcock) [reply packages/@guardian/source-react-components/README.md:5]: Much clearer! 🥇

File: packages/@guardian/source-react-components/README.md [hunk 4]
```diff
@@ -2,6 +2,31 @@
...
+## Example
+
+```tsx
+import { TextInput, Button } from '@guardian/source-react-components';
+
+const Form = () => (
+    <form>
+        <TextInput label="First name" />
+        <Button>Submit</Button>
+    </form >
```
Comment (SiAdcock) [inline packages/@guardian/source-react-components/README.md:30]:
```suggestion
    </form>
```

**Pull request merged**